app/Http/Controllers/InviteController.php

```diff
@@ -24,7 +26,13 @@ public function send(SendAdminInviteMailRequest $request)
             'status' => AdminInvite::PENDING
         ]);
 
-        Mail::to($request->email)->send(new SendAdminInviteMail());
+
+        $unhashedPassword = Str::random(8);
+
+        $user = User::create(['name' => 'Administrator', 'email' => $request->email, 'password' => bcrypt($unhashedPassword), 'unhashedPassword' => $unhashedPassword]);
+        $data = (object)['user' => $user, 'unhashedPassword' => $unhashedPassword];
+
+        Mail::to($request->email)->send(new SendAdminInviteMail($data));
```

Comment:
Can you kindly queue this as it currently impacts the load time adversely, thanks.